doc/README_Generic.md

```diff
@@ -0,0 +1,61 @@
+Generic YaST README File
+========================
+
+YaST Development Documentation
+------------------------------
+
+- [YaST development documentation](http://yast.github.io/documentation.html)
+- [Contribution Guidelines](http://yast.github.io/guidelines.html)
+- [YaST architecture](http://yastgithubio.readthedocs.org/en/latest/architecture)
+- [Development environment](https://en.opensuse.org/openSUSE:YaST:_Preparing_the_Development_Environment)
+
+
+Development Environment
+-----------------------
+Before doing anything useful with the code, you need to setup a development
+environment. Fortunately, this is quite simple, see the [preparing development
+environment](https://en.opensuse.org/openSUSE:YaST:_Preparing_the_Development_Environment)
+documentation.
+
+
+### Extra Development Tools ###
```

Comment:
I wouldn't add any extra information here, but rather put the wiki page in a better shape. The information is actually missing there.

In fact, I was considering to move that page from the wiki to the github.io repo (like the document about architecture or the development tips&tricks). Right now most user oriented pages are in the wiki and most developer oriented ones are in yast.github.io/doc, so...